code/modules/surgery/organs/lavaland.dm

```diff
@@ -82,6 +82,14 @@
 		if(owner.mind.assigned_role == "Mime")
 			power_multiplier *= 0.5
 
+	//Cultists are closer to their gods and are more powerful, but they'll give themselves away
+	if(iscultist(owner))
+		power_multiplier *= 2
+		spans = list("narsie")
```

Comment:
You might want to make a new pair of spans for these; the colossus span is size 5, while narsie is size 15 and ratvar is size 6.
Also, reset the spans at some point, so if someone surgeries it out and uses it it doesn't stay like this.